init-project/db-user-schema-create.sql

```diff
@@ -66,6 +83,19 @@ CREATE TABLE `user` (
   CONSTRAINT `FK_077f495e15f6c189c728fc5d2f3` FOREIGN KEY (`businessAccountId`) REFERENCES `business_account` (`id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
 
+--
+-- Table structure for table `brand_status_history`
+--
+
+DROP TABLE IF EXISTS `brand_status_history`;
+CREATE TABLE `brand_status_history` (
+  `id` varchar(255) NOT NULL AUTO_INCREMENT,
```

Comment:
AUTO_INCREMENT needs to be removed to be able to execute sql script successfully